### DISCUSSION TITLE
New Source: Redis

### DISCUSSION THREAD

**Discussion by mannharleen** [airbytehq/airbyte#36107] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 

Redis source. I am particularly interested in redis stream but I believe other types will also be valuable.

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
* How often do you want to run syncs?
* If this is an API source connector, which entities/endpoints do you need supported?
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 

Redis (stream) data needs to be synced to a) warehouse b) kafak for analysis.

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?

If redis streams source is not available, I might have to use an alternative tool like Benthos

## Are you willing to submit a PR?

Sorry I wish I could but I am not proficient in python.

**Comment by marcosmarxm** [comment]
Adding this to `team/db-dw-sources` because Redis has JDBC driver.

**Comment by charity1475** [comment]
Is the Redis source done or in the current development stream?  Should I start development or it's a duplicate work ? @marcosmarxm

[a comment by an automated account is not shown]
